src/Plugin/DataType/StrawberryValuesViaJmesPathFromJson.php

```diff
@@ -85,6 +85,11 @@ public function process($langcode = NULL)
       }
       // This is an array, don't double nest to make the normalizer happy.
       $jmespath_result_to_expose = array_filter($jmespath_result_to_expose);
+      foreach($jmespath_result_to_expose as $i => $v) {
+        if(!is_string($v)) {
```

Comment:
May be also a number friend... Do we want to check for (is_array(..) || is_object(..)) instead?

Reply:
I think it may only be array since we json_decoded but just to be sure ...